### ISSUE TITLE
Blender template doesn't work outside of the Character Creator

### ISSUE THREAD

**Issue by DBird-1994** [TheGameCreators/GameGuruRepo#4814]
When using the new Blender templates, exporting the character parts for use in the CC results in a 100% working character as shown in the below video:

https://github.com/TheGameCreators/GameGuruRepo/assets/136038988/756d088f-cf0f-4887-9b67-040d95db0617

However, when the same character is exported from Blender as an entire character, then imported into MAX as a new, non-CC character,  it will freeze after being shot by the player as seen below.    I'm not sure if this is a bug or is by design, but it would be nice to use the Blender template to create new characters that can be used straightaway, rather than peiced together inside the CC.

https://github.com/TheGameCreators/GameGuruRepo/assets/136038988/9caabf47-b3cd-484f-af69-49f745928692

**Comment by TenebreGaming** [comment]
I have encountered the same issue and agree that it should work 100% as a full model not chopped into pieces, as that just takes a lot more time and seems quite unnecessary.

**Comment by bond1mjb** [comment]
@TenebreGaming @DBird-1994 I can replicate this problem on my end too.   And this not only happens with the new FBX templates, but also with full characters exported directly from 3ds Max using the older .X format.    So unless we're missing an import setting somewhere, it looks like GGMAX's built in character behaviors only work with characters assembled in the CC.

I would agree that it would be very useful to have the behaviors work with characters outside of the CC.  I can't really see a reason for them not  to work since they share the same skeleton.   Especially since they seem to work fine right up until the point you shoot them.    I'll have to let Lee handle this one since I really don't know what is going on here.

**Comment by DBird-1994** [comment]
@TenebreGaming As a workaround, I created a set of BLANK body parts for the CC,  so that you can use your full character inside the CC without having to first chop up your character in Blender. Just set the head, lets, and feet to NONE then use your custom character as the body.  Works great and the character no longer freezes when shot.

Big thanks to @bond1mjb for help with this!

[EmptyCCParts.zip](https://github.com/TheGameCreators/GameGuruRepo/files/12207613/EmptyCCParts.zip)

https://github.com/TheGameCreators/GameGuruRepo/assets/136038988/233f5779-66d9-41c3-a7d5-fdd6cf31cd7e

**Comment by LeeBamberTGC** [comment]
@bond1mjb @TenebreGaming May I use these blank parts and video to add to the Characters Guide officially?  A sumer nice workaround for now.  I also invite some ideas how we can make this a real feature of the character creator, i.e. a large button titled "Import Full Character" that will automatically hide the unneeded parts, but otherwise put the third party character through the same ringer to produce a fully MAX compatible character for the behaviours.

**Comment by bond1mjb** [comment]
@LeeBamberTGC It's fine with me, but it was @DBird-1994 who exported these from the Blender template.    They are zero-geometry versions of the character parts.   I've attached  some empty versions for the female as well.

I agree they work well as a workaround for importing full characters, but having empty parts included in the CC could come in useful for other types of characters down the line that don't need certain body parts like ghosts and other supernatural type enemies.    But I like the idea of having a button called "Import Full Character" too.
[empytcc parts female.zip](https://github.com/TheGameCreators/GameGuruRepo/files/12209866/empytcc.parts.female.zip)

**Comment by LeeBamberTGC** [comment]
@DBird-1994 Sorry for not giving credit!  Are you okay with your blank parts going into the official build?

@bond1mjb Many thanks for the female set, I think that should do it as a workaround until the new 'import cool character' button is added :)

**Comment by DBird-1994** [comment]
@LeeBamberTGC Of course, I would be honored if you want to include the blank parts in the official build!
